core/src/test/java/org/infinispan/partitionhandling/NumOwnersNodeCrashInSequenceTest.java

```diff
@@ -56,7 +55,7 @@ protected void createCacheManagers() throws Throwable {
       configBuilder.clustering().hash().numSegments(4).stateTransfer().timeout(30000);
       globalBuilder = GlobalConfigurationBuilder.defaultClusteredBuilder();
       // Must explicitly add with id >= 2500 so that the user marshaller is able to store it
-      globalBuilder.serialization().addAdvancedExternalizer(JBossUserMarshaller.USER_EXT_ID_MIN, new JGroupsAddress.Externalizer());
+      globalBuilder.serialization().addAdvancedExternalizer(2500, new JGroupsAddress.Externalizer());
```

Comment:
We should probably store the min somewhere again still, so we can reference if needed.

Reply:
The `ControlledConsistentHashFactory.Default` now has it's own Externalizer, so no need fo rthis anyway. Any ideas on where to store the value?

Reply:
Unless we put it in the `AdvancedExternalizer` interface?

Reply:
Seems like as fine of a place as any.